neo/Prompt/Commands/Show.py

```diff
@@ -0,0 +1,159 @@
+import os
+import psutil
+import datetime
+from neo.Prompt.CommandBase import CommandBase, CommandDesc, ParameterDesc
+from neo.Prompt.PromptData import PromptData
+from neo.Prompt.Utils import get_arg
+from neo.Core.Blockchain import Blockchain
+from neocore.UInt256 import UInt256
+from neo.IO.MemoryStream import StreamManager
+from neo.Network.NodeLeader import NodeLeader
+from neo.Implementations.Notifications.LevelDB.NotificationDB import NotificationDB
+from neo.logging import log_manager
+import json
+
+
+logger = log_manager.getLogger()
+
+
+class CommandShow(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+        self.register_sub_command(CommandShowBlock())
+        self.register_sub_command(CommandShowHeader())
+        self.register_sub_command(CommandShowTx())
+        self.register_sub_command(CommandShowMem())
+        self.register_sub_command(CommandShowNodes(), ['node'])
+
+    def command_desc(self):
+        return CommandDesc('show', 'show data from the blockchain')
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+
+        if not item:
+            print("run `%s help` to see supported queries" % CommandShow().command_desc().command)
+            return
+
+        try:
+            return self.execute_sub_command(item, arguments[1:])
+        except KeyError:
+            print(f"{item} is an invalid parameter")
+            return
+
+
+class CommandShowBlock(CommandBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
```

Comment:
`show block` and `show headers` without an argument throws an exception. Example
```
neo> show block
Could not execute command: object of type 'NoneType' has no len()
and then some
```

`show tx` without arguments throws a different exception
```
neo> show tx
Could not find transaction from args: 'NoneType' object is not subscriptable ([])
```

Reply:
I removed all those user friendly messages because I thought @LysanderGG 's change to CommandBase here https://github.com/CityOfZion/neo-python/pull/733#pullrequestreview-181093593 was going to be merged. Oops 😊 
I will add all the original messages back in